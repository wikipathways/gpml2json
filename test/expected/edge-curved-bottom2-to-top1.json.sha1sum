d0930a7d9acaa6be7ae4b6780d5521ed5fef0549 ?-
