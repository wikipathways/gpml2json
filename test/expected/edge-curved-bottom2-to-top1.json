{"pathway":{"@context":["https://cdn.rawgit.com/wikipathways/WpVocabularies/7a46a05/contexts/pvjs.jsonld"],"backgroundColor":"white","borderWidth":0,"color":"black","contains":["ca8eb","dd71a","bece9"],"drawAs":"rect","gpmlElementName":"Pathway","height":783.1967827425314,"kaavioType":"Group","name":"test case","padding":5,"type":["Pathway"],"width":903.4992514498286,"x":0,"y":0,"zIndex":0,"fontSize":12,"fontWeight":"bold","textAlign":"left","verticalAlign":"top","dataSourceVersion":"20130813","standardName":"test case","displayName":"test case","textContent":"Name: test case","isSimilarTo":"http://wikipathways.org/index.php/Special:SearchPathways?query=test%20case&doSearch=1","id":"hash://sha256/6490dd8cf662551e28647b92e2f57e8d664e14e3042a9a083c8cc78e44b1e2e8"},"entityMap":{"dd71a":{"id":"dd71a","textContent":"GeneProduct","wpType":"GeneProduct","x":119.84985784453619,"y":155.46800655730547,"fontSize":10,"height":20,"verticalAlign":"middle","width":80,"zIndex":33008,"textAlign":"center","color":"#000000","backgroundColor":"#ffffff","textDecoration":"none","fontFamily":"Arial","fontStyle":"normal","fontWeight":"normal","borderWidth":1,"drawAs":"rect","lineHeight":1.1,"overflow":"hidden","textOverflow":"clip","whiteSpace":"pre","padding":8,"type":["DataNode","SingleFreeNode","GeneProduct"],"kaavioType":"SingleFreeNode","gpmlElementName":"DataNode"},"bece9":{"id":"bece9","textContent":"0022","wpType":"GeneProduct","x":27.365970667488995,"y":62.952101682289026,"fontSize":10,"height":20,"verticalAlign":"middle","width":80,"zIndex":33009,"textAlign":"center","color":"#000000","backgroundColor":"#ffffff","textDecoration":"none","fontFamily":"Arial","fontStyle":"normal","fontWeight":"normal","borderWidth":1,"drawAs":"rect","lineHeight":1.1,"overflow":"hidden","textOverflow":"clip","whiteSpace":"pre","padding":8,"type":["DataNode","SingleFreeNode","GeneProduct"],"kaavioType":"SingleFreeNode","gpmlElementName":"DataNode"},"ca8eb":{"id":"ca8eb","drawAs":"CurvedLine","borderWidth":1,"zIndex":33007,"points":[{"isAttachedTo":"bece9","attachmentDisplay":{"position":[0.25,1],"offset":[0,0]},"x":47.365970667488995,"y":82.95210168228903,"orientation":[0,1]},{"x":103.60791425601259,"y":102.95210168228903},{"isAttachedTo":"dd71a","attachmentDisplay":{"position":[0.5,0],"offset":[0,0]},"x":159.8498578445362,"y":155.46800655730547,"orientation":[0,1]}],"color":"#000000","type":["Interaction","Edge"],"kaavioType":"Edge","gpmlElementName":"Interaction","markerStart":"TBar","markerEnd":"Arrow","isAttachedTo":["bece9","dd71a"]}}}
